### ISSUE TITLE
Swagger removing html attributes from description

### ISSUE THREAD

**Issue by ocdevops** [swagger-api/swagger-ui#3225]
Swagger version:3.0.14

The new Swagger version is removing the html attributes from the description.
Original in the index.json file:
```
"info": {
    "title": "Documentation",
    "description": "<table><tr><td colspan=\"3\"><span class=\"method\">PUT</span></td></tr></table>",
    "contact": ""
  },
```

Cleared  version:
```
"info": {
    "title": "Documentation",
    "description": "<table><tr><td><span>PUT</span></td></tr></table>",
    "contact": ""
  },
```

**Comment by webron** [comment]
This is basically for the same reason as #3224.

**Comment by Frank591** [comment]
Does any workaround exist to fix this behaviour? I need to do links to local anchors.
Example:
```
description: | 
<a href="#04.02.2018">Fast link to 04.02.2018</href>
... a lot of text here...
<div id="04.02.2018"><h3>04.02.2018</h3></div>
```

**Comment by heldersepu** [comment]
@Frank591 I don't think that `<div id="04.02.2018">` is a proper anchor...
shouldn't that be:
`<h3><a name="04.02.2018">04.02.2018</a></h3>`

**Comment by Frank591** [comment]
@heldersepu , thank you! It works.

**Comment by heldersepu** [comment]
Yes @Frank591 you are correct the comment from @ocdevops still valid... 
But it the UI had a few release since 3.0.14, I will test with latest let's see

**Comment by Frank591** [comment]
@heldersepu , I tested with swagger-ui 3.9.3, and it still not working.

**Comment by heldersepu** [comment]
Yep it wiped out the colspan:
http://swashbuckletest.azurewebsites.net/swagger/ui/index?filter=Big#/BigString/BigString_Get

**Comment by heldersepu** [comment]
I think the Markdown Sanitizer need some improvements...
That is probably very very low priority

**Comment by Frank591** [comment]
Yes, i think, you are right. i got solution for my local problem. Will leave this issue in open state, may be it will be fixed in the future.

**Comment by Frank591** [comment]
@heldersepu , thanks for help!

**Comment by heldersepu** [comment]
This could be a starting point:
https://www.npmjs.com/package/sanitize-html#wildcards-for-attributes
The code seems to be here:
https://github.com/swagger-api/swagger-ui/blob/master/src/core/components/providers/markdown.jsx

A relevant commit:
https://github.com/swagger-api/swagger-ui/commit/60e8091eedde05244a1d8c498b5f85f7390ed9af

**Comment by shockey** [comment]
This specific use case is fixed in #4194 🎉
